### ISSUE TITLE
16 and 24 gb cards

### ISSUE THREAD

**Issue by IIIOXIII** [develsoftware/GMinerRelease#122]
Not a specific issue, however, I haven't seen any of the miners take advantage of the 16/24 increased memory sizes on the new card versions.  Is this possible?  With the lack of stock on some of the lower memory modules, it would be nice to get more performance out of the higher end cards.

I have no idea about hashing algorithms but have written a lot of code in the past.  Would be happy to help if interested.  Just trying maximize OUR profits!

Thanks again for your work.

**Comment by Rickytr23** [comment]
Yes I sounds good. I want to get 16/24 card to be ready for the future.
